pkg/crc/gpg/gpg.go

```diff
@@ -0,0 +1,96 @@
+package gpg
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/ProtonMail/go-crypto/openpgp"
+	"github.com/ProtonMail/go-crypto/openpgp/armor"
+	"github.com/ProtonMail/go-crypto/openpgp/packet"
+	"github.com/code-ready/crc/pkg/crc/constants"
+)
+
+func readSignature(signatureFilePath string) (*packet.Signature, error) {
+	sigFile, err := os.Open(signatureFilePath)
+	if err != nil {
+		return nil, err
+	}
+	sigFileBlock, err := armor.Decode(sigFile)
```

Comment:
https://github.com/code-ready/crc/pull/3221#discussion_r913570665

Reply:
@cfergeau I have updated to `ProtonMail/go-crypto` what is missing now?